src/components/ui_parts/card/phoneCard.php

```diff
@@ -0,0 +1,21 @@
+<?php /*
+  概要: スマホ用Cardパーツ
+    title: string
+    description: string
+    image: string
+*/ ?>
+
+<div class="mx-4 rounded-2xl border flex flex-col gap-4 items-center overflow-hidden ">
```

Comment:
[must]
背景色を白に設定しないと裏のロゴが透けて見えてしまう。

```suggestion
<div class="mx-4 rounded-2xl border flex flex-col gap-4 items-center overflow-hidden bg-white">
```